examples/servicegraph/cmd/server/BUILD

```diff
@@ -0,0 +1,21 @@
+package(default_visibility = ["//visibility:private"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_binary")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "main.go",
+    ],
+    deps = [
+        "//examples/servicegraph:go_default_library",
+        "//examples/servicegraph/dot:go_default_library",
+        "//examples/servicegraph/promgen:go_default_library",
+    ],
+)
+
+go_binary(
+    name = "servicegraph",
+    data = ["//examples/servicegraph:js/viz-lite.js"],
```

Comment:
possibly. it isn't really needed anywhere in general (exists entirely for visual debugging purposes in early days). when we settle on the right place to put everything, we can just delete it entirely.